frontend/src/components/BlockDetails/BlockDetailsTokenomics.vue

```diff
@@ -17,7 +17,19 @@
 			/>
 
 			<FinalizationRewards
-				v-if="data.block.finalizationRewards.nodes.length"
+				v-if="
+					data.block.finalizationRewards.nodes.length &&
+					data.block.finalizationRewards.nodes.reduce((acc, current) => {
```

Comment:
I think this should probably be a function so that it becomes more obvious why this is here and what it does.